packages/notifications/src/toast/ToastContainer.tsx

```diff
@@ -14,10 +14,10 @@ const ToastContainer = styled.div<any>(
       marginBottom: theme.space.default,
       minWidth: rem(180),
       maxWidth: `calc(100vw - ${theme.space.small * 2}px)`,
-      [mq(theme.breakpoint.medium)]: {
+      [media(theme.newTokens.breakpoint.medium)]: {
```

Comment:
I think that notification we can leave on old tokens, that one is legacy component

Reply:
Reverted